client/my-sites/post-type-list/post-actions-ellipsis-menu/view.jsx

```diff
@@ -42,6 +42,7 @@ class PostActionsEllipsisMenuView extends Component {
 
 		this.props.setPreviewUrl( previewUrl );
 		this.props.setLayoutFocus( 'preview' );
+		this.props.onClick();
```

Comment:
Based on the [behavior of `<PopoverMenu />`](https://github.com/Automattic/wp-calypso/blob/4d7c43ef15b7b6a0dd75bc3e3f3b29bb8fa51782/client/components/popover/menu.jsx#L55-L70), I wouldn't expect that you'd need to call `onClick` explicitly like this. Why isn't the `onClick` handler from this method being invoked automatically? At the very least we should define the prop as required or specify a default noop, but still feels strange that we need this.